djangocms_content_expiry/templates/admin/change_list.html

```diff
@@ -0,0 +1,9 @@
+{% extends "admin/change_list.html" %}
+{% load i18n %}
+{% load admin_urls %}
+{% block object-tools-items %}
+    {{ block.super }}
+    <li>
+        <a class="historylink" href="{% url opts|admin_urlname:'export_csv' %}{{cl.get_query_string}}">Export</a>
```

Comment:
All text must be multilingual because our users require it. 

```suggestion
        <a class="historylink" href="{% url opts|admin_urlname:'export_csv' %}{{cl.get_query_string}}">{% translate "Export" %}</a>
```